syntaxes/qcpu2ci.tmLanguage.json

```diff
@@ -5,10 +5,9 @@
 	"patterns": [
 		{ "include": "source.qcpu2c#comment" },
 		{ "include": "source.qcpu2c#label" },
-		{ "include": "source.qcpu2c#aliases" },
+		{ "include": "#aliases" },
 		{ "include": "source.qcpu2c#operators" },
 		{ "include": "#assembly" },
-		{ "include": "source.qcpu2c#assembly?" },
```

Comment:
Are you sure about removing `source.qcpu2c#assembly?`?
it was so instructions aren't red while you type them

Reply:
Ah, I thought it was an optional import from the other language configuration. I'll fix it in the morning